src/utils/decorators.ts

```diff
@@ -48,6 +48,11 @@ export function createClassPropertyDecorator(
 					}
 				}
 			};
+			if (arguments.length < 3) {
+				// Typescript target is ES3, so it won't define property for us
+				Object.defineProperty(target, key, descriptor);
```

Comment:
I mean some people can forget to specify typescript target, i thought, it would be nice if mobx still works. Fixing jsfiddle is also nice.
And it fixes Reflect.decorate (#333)